packages/jest-worker/src/Farm.ts

```diff
@@ -147,16 +151,17 @@ export default class Farm {
   private _push(task: QueueChildMessage): Farm {
     this._taskQueue.enqueue(task);
 
+    const offset =
+      this._workerSchedulingPolicy === 'round-robin' ? this._offset++ : 0;
```

Comment:
sorta hard to follow the logic with conditional incrementation - pull it out into an `if` statement?